docs/manual/forms/examples/camunda7.rst

```diff
@@ -58,7 +58,7 @@ Formulier maken
 #. Voeg een actie aan de regel toe, van type **Evalueer DMN** en klik op de **Instellen** knop.
```

Comment:
It's not shown in the changes, but in the first step of `Formulier maken` it says `nummer component`, while it is called `Getal` in the form builder, did the translation change sometime in the past?

Reply:
I don't think so, but I'm changing the docs!